### ISSUE TITLE
Adventure Landing Page

### ISSUE THREAD

**Issue by KRRISHSINGH08** [0silverback0/LandingPages#75]
I want to create a new landing page.

**Comment by 0silverback0** [comment]
in the landing page root directory in index.html add your information

**Comment by 0silverback0** [comment]
this is in the project folder but not in the  /index.html

**Comment by KRRISHSINGH08** [comment]
Yes, i accidentally pushed it with some other pull request i opened. Can you help me ? What should i do if i make another pull request then there will be 2 project of same name.  here #64

**Comment by 0silverback0** [comment]
no its ok i believe its already added to the cards but if you want to make a change to it and then make a PR that works too

**Comment by KRRISHSINGH08** [comment]
Is it possible to delete the old adventure file i added and merge this one here #99
